rocAL/rocAL/source/decoders/image/fused_crop_decoder.cpp

```diff
@@ -87,21 +74,18 @@ Decoder::Status FusedCropTJDecoder::decode(unsigned char *input_buffer, size_t i
     // check the vector size for bounding box. If its more than zero go for random bbox crop
     // else go to random crop
     unsigned int crop_width, crop_height, x1, y1, x1_diff, crop_width_diff;
-    if(_bbox_coord.size() != 0)
-    {
+    if(_bbox_coord.size() != 0) {
         // Random bbox crop returns normalized crop cordinates
         // hence bringing it back to absolute cordinates 
         x1 = std::lround(_bbox_coord[0] * original_image_width);
         y1 = std::lround(_bbox_coord[1] * original_image_height);
         crop_width = std::lround((_bbox_coord[2]) * original_image_width);
         crop_height = std::lround((_bbox_coord[3]) * original_image_height);
     }
-    else
-    {
+    else {
```

Comment:
Else to be moved to previous line
`} else {`
Check the google coding guidelines

Reply:
Changed